src/Dropdown.php

```diff
@@ -295,6 +295,19 @@ private function renderItems(array $items, array $itemsOptions = []): string
                 $label = $item['label'];
             }
 
+            $icon = '';
+            $iconOptions = [];
+
+            if (isset($item['icon'])) {
```

Comment:
You can do `$icon = $item['icon'] ?? ''` instead.

Reply:
done.